src/models/user.ts

```diff
@@ -283,8 +283,8 @@ export class User extends SubscribableModel {
     return new UserFacilityEmployeeRelationships(facilityRelationships, facilityRelationshipsMeta, this.sessionHandler)
   }
 
-  async createDevelopmentAccount (options: { company?: string, address?: string, websiteUrl?: string }) {
-    const { developmentAccount } = (await this.action('developmentAccount:create', { ...options })) as DevelopmentAccountResponse
+  async createDevelopmentAccount (params: { company: string, address: string, websiteUrl: string, privacyUrl: string, termsUrl: string }) {
```

Comment:
Let's revert `company`, `address`, and `websiteUrl` to be optional again.  Just so it matches what is currently supported on the API.